packages/@adobe/spectrum-css-temp/components/tags/index.css

```diff
@@ -69,4 +70,19 @@ governing permissions and limitations under the License.
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
+  outline: none;
+}
+
+.spectrum-Tag-action {
+  grid-area: action;
+  height: 75%;
+  width: var(--spectrum-global-dimension-size-100);
+  padding-top: var(--spectrum-global-dimension-size-25);
+  padding-inline-start: var(--spectrum-global-dimension-size-100);
+
+  &:focus-ring {
+    border-color: var(--spectrum-dropzone-border-color-selected-hover);
+    box-shadow: 0 0 0 1px var(--spectrum-dropzone-border-color-selected-hover);
+    border-radius: 20%;
```

Comment:
where'd 20% come from? usually we use a pixel value, because percent is based on both height and width, which may change from tag to tag

Reply:
it's for the focus ring of the remove 'x'. i can definitely change it use a pixels instead